x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/hosts/kpi/unique_ips/index.ts

```diff
@@ -11,17 +11,18 @@ import type { IEsSearchResponse } from '@kbn/data-plugin/common';
 import type {
   HostsKpiQueries,
   HostsKpiUniqueIpsStrategyResponse,
-  HostsKpiUniqueIpsRequestOptions,
 } from '../../../../../../../common/search_strategy/security_solution/hosts';
 import { inspectStringifyObject } from '../../../../../../utils/build_query';
 import type { SecuritySolutionFactory } from '../../../types';
 import { buildHostsKpiUniqueIpsQuery } from './query.hosts_kpi_unique_ips.dsl';
 import { formatGeneralHistogramData } from '../../../common/format_general_histogram_data';
 
 export const hostsKpiUniqueIps: SecuritySolutionFactory<HostsKpiQueries.kpiUniqueIps> = {
-  buildDsl: (options: HostsKpiUniqueIpsRequestOptions) => buildHostsKpiUniqueIpsQuery(options),
+  buildDsl: (options) => {
+    return buildHostsKpiUniqueIpsQuery(options);
+  },
```

Comment:
Or is there a reason why we would prefer this syntax here? Debugging?

Reply:
yes, I will make it simpler.